metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache License, Version 2.0'
 description 'Installs/Configures chef-dash'
 long_description 'Installs/Configures chef-dash'
-version '0.5.0'
+version '0.4.4'
```

Comment:
Hi Daniel, It is purposely on 0.4 series because once the PR is merged, Pingworks should make it 0.5.1.  I do not want to make it so myself until they merge the PR.